packages/storybook8/stories/Composites/CallWithChatComposite/Docs.mdx

```diff
@@ -0,0 +1,195 @@
+details.
+
+## Adding file sharing
+
+### File Sharing In Azure Communication Services Chat Thread
+
+<SingleLineBetaBanner />
+
+In a chat thread where all participants are Azure Communication Services end users, the `CallWithChat`
+Composite supports file capabilities in conjunction with your choice of a storage solution. Using the provided
+APIs, you can composite to support uploading files and displaying them on the send box before sending, and the
+once they have been sent or received. For an end to end tutorial on enabling file sharing with Azure Blob
+Storage as your choice of a storage solution, refer to our
+[tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/file-sharing-tutorial).
+
+<Source
+  code={`<CallWithChatComposite
+adapter={chatAdapter}
+options={{
+  attachmentOptions: {
+    uploadOptions: {
+      supportedMediaTypes: ['image/png', 'image/jpeg'],
+      disableMultipleUploads: false,
+      handler: attachmentUploadHandler
+    },
+    /* If downloadOptions is not provided. The file URL is opened in a new tab.
+    You can find examples of downloadOptions and uploadOptions in this tutorial
+    https://docs.microsoft.com/en-us/azure/communication-services/tutorials/file-sharing-tutorial */
+    downloadOptions: {
+      actionsForAttachment: (attachment: AttachmentMetadata, message?: ChatMessage): AttachmentMenuAction[] => {
+        return [
+          defaultAttachmentMenuAction,
+          {
+            name: 'Open',
+            icon: <WindowNew24Regular />,
+            onClick: () => {
+              return new Promise((resolve) => {
+                window.alert('open button clicked');
+                resolve();
+              });
+            }
+          }
+        ];
+      }
+    }
+  }
+}} />`}
+/>
+
+### File Sharing In Teams Interop Meeting Chat Thread
+
+When using the `CallWithChat` Composite in a Teams interop scenario, the Azure Communication Services end user
+can recieve file attachments from Teams user without any additional setup. Simply join the Teams meeting as an
+Azure Communication Services end user, as soon as the Teams user sends a file from the Teams client, the Azure
+Communication Services end user will be able to see shared files in the chat thread.
+
+## PSTN and 1:N Calling
+
+<SingleLineBetaBanner />
+<MessageBar>Note: see CallComposite for detailed implementation description</MessageBar>
+
+The CallWithChatComposite supports the PSTN Calling modality as well. With CallWithChat you will need to still
+provide a chat thread Id for instances where you will add a Azure Communications user to the call. If you are
+looking to use the CallWithChatComposite and are looking to hide the chat because you are only calling PSTN
+users please see our documentation on how to hide the [control bar
+buttons](./?path=/story/composites-call-with-chat-basicexample--basic-example).
```

Comment:
```suggestion
buttons](./?path=/story/composites-call-with-chat-basicexample-basic-example--basic-example).
```

Reply:
thanks for checking on this. Ill fix this up

Reply:
updated with your suggestions.